src/style.css

```diff
@@ -0,0 +1,190 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.main {
+  width: 80%;
+  margin-left: 10%;
+  border: 4px solid black;
+}
+
+.white {
+  color: white;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 12px;
+  border: 2px solid black;
+  height: 40px;
+  margin-bottom: 80px;
+}
+
+.top-nav {
+  background: #6e7072;
+  border: 2px solid black;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.top-nav > * {
+  box-sizing: border-box;
+}
+
+.botom-nav {
+  padding: 0 20px;
+}
+
+.top-part {
+  text-align: center;
+}
+
+.bottom-part {
+  display: flex;
+  align-items: center;
+  margin-bottom: 20px;
+  padding-bottom: 20px;
+  justify-content: space-around;
+}
+
+.bottom-part p {
+  color: #fff;
+  font-size: 30px;
+  cursor: pointer;
+  margin-left: 20px;
+}
+
+.http-input {
+  border: 2px solid rgb(31, 28, 28);
+  border-radius: 8px;
+  margin-left: 80px;
+  height: 40px;
+  padding-left: 5px;
+  width: 40%;
+  font-size: 1rem;
+}
+
+.curved-input {
+  width: 80px;
+  height: 40px;
+  border: 2px solid black;
+  border-radius: 10px;
+  margin-left: 40px;
+}
+
+.search-bar {
+  display: flex;
+  height: 40px;
+}
+
+#search-text-input {
+  border: 2px solid black;
+  height: 40px;
+  width: 80px;
+  border-bottom-right-radius: 0;
+  border-top-right-radius: 0;
+  padding: 4px;
+  font-size: 1rem;
+}
+
+.fa-magnifying-glass {
+  background-color: rgb(226, 226, 226);
+  border-top-right-radius: 10px;
+  border-bottom-right-radius: 10px;
+  border-bottom: 2px solid black;
+  border-right: 2px solid black;
+  border-top: 2px solid black;
+  padding: 10px 6px;
+  width: 28px;
+}
+
+.leader {
+  margin-left: 12px;
+}
+
+.container {
+  display: flex;
+  margin-left: 10%;
+  margin-bottom: 40px;
+}
+
+.main__result-top {
+  display: flex;
+  align-items: center;
+  margin-bottom: 10px;
+}
+
+.refresh {
+  width: 60px;
+  height: 30px;
+  margin-left: 16px;
+  border-right: 4px solid black;
+  border-bottom: 4px solid black;
+}
+
+.main__result-bottom {
+  border: 4px solid black;
+}
+
+.name-score {
+  list-style: none;
+  margin-left: -30px;
+}
+
+.name-score li {
+  margin-bottom: 12px;
+}
+
+.name-score :nth-child(even) {
+  background: rgb(211, 205, 205);
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.main__input {
+  display: flex;
+  flex-direction: column;
+  margin-left: 60px;
+}
+
+.input-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.input-form input {
+  margin-bottom: 20px;
+  height: 40px;
+  border: 4px solid black;
+  font-size: 20px;
+  font-weight: 600;
+}
+
+.input-form button {
+  width: 60px;
+  margin-left: 20px;
+  align-self: flex-end;
+  margin-top: 12px;
+  border-bottom: 4px solid black;
+  border-right: 4px solid black;
+}
+
+.footer {
+  width: 80%;
+  margin-left: 10%;
+  border: 4px solid black;
+  background: rgb(180, 171, 171);
+}
+
+.footer p {
+  display: flex;
```

Comment:
- [ ] Great job working on this milestone. We have to try as much as possible to follow the [DRY principles.](https://vanseodesign.com/css/dry-principles/). When I look at your CSS, I find so many `display: flex` . You can use this just once in your CSS and pass flex in your html as a class like this `<div class="container flex">`
```
.flex{
   display: flex;
       }
```